.github/workflows/build.yml

```diff
@@ -111,6 +111,8 @@ jobs:
       - test-linux-arm64
       - test-windows
     runs-on: ubuntu-latest
+    permissions:
+      id-token: write
```

Comment:
Needed to request an OIDC token: https://docs.github.com/en/actions/deployment/security-hardening-your-deployments/about-security-hardening-with-openid-connect#adding-permissions-settings